### DISCUSSION TITLE
Make Site public within organization only?

### DISCUSSION THREAD

**Discussion by Oobiewan** [community/community#70627] [Enterprise]
### Pages site visibility
Is there a way to make the site accessible to everyone within the organization?

We have GitHub Enterprise and have a private repository with some documentation source that we'd like to publish as a site for anyone within the organization. But as I understand, if we set the visibility of the page to "private", only those will have access to the site who also have access to the repository. If we set the visibility to "public", the site will be accessible to everyone on the internet. Is there a way to make the site accessible to everyone within the organization without also giving them access to the repository?

**Comment by BlueSpider0130** [comment]
Thank you
